### PR TITLE
Portfolio Desktop Version

### PR DESCRIPTION
**Pull request by ProgramKingAbel**
-Configured Linters (WebHint, Stylelint, Lighthouse)
-Improved responsiveness - adjusted container wrapper width percentages.
-Added Media Queries for desktop (768px breakpoint)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -407,3 +427,108 @@ button[type="submit"]:hover {
...
+  }
+
+  .switched-cards-info {
+    padding-top: 1rem;
+    padding-left: 1rem;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (LionRouge1) [inline styles.css:482]: - [ ] Nice job, your design look almost identic to Figma's design. :muscle: :sunglasses: However in order to make the "About Myself" section look exactly as the Figma design (see image), kindly make the left block takes 40% and the right block the remaining 60% of the page's width. You can rewrite this line and use `grid-template-columns: 40% 60%;`
### Figma's Design
Screenshot from 2023-04-05 06-38-27

### Your Design
Screenshot from 2023-04-05 06-56-44

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @ProgramKingAbel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- You implement almost all the requirements:+1: 
- Your design looks 98% identic to Figma's Design :1st_place_medal: :sunglasses: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] Good job adding all the main headers from the provided [README template](https://github.com/microverseinc/readme-template) :clap: :sunglasses: However the "Built With" section is empty, please describe the tech stack that you used to build your project in this section. That will make your README file looks more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (ProgramKingAbel) [reply styles.css]: Thank you for the guidance. I have implemented the changes as advised.

**Review by christianbiring1 — APPROVED** [review]
Hi @ProgramKingAbel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] ALL LINTERC CHECKS ARE GREEN :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**